Tests/Remora.Results.Tests/ResultTests.cs

```diff
@@ -389,4 +389,41 @@ public void CreatesAnUnsuccessfulResultFromAnExceptionInstance()
             Assert.IsType<ExceptionError>(result.Error);
         }
     }
+
+    /// <summary>
+    /// Tests the <see cref="Result.SuccessfulResult"/> property.
+    /// </summary>
+    public class SuccessfulResult
+    {
+        /// <summary>
+        /// Tests whether <see cref="Result.SuccessfulResult"/> returns a successful result.
+        /// </summary>
+        [Fact]
+        public void ReturnsASuccessfulResult()
+        {
+            var successful = Result.SuccessfulResult;
+            Assert.True(successful.IsSuccess);
+        }
+
+        /// <summary>
+        /// Tests whether <see cref="Result.SuccessfulResult"/> returns a result whose inner property is null.
+        /// </summary>
+        [Fact]
+        public void ReturnsAResultWhoseInnerPropertyIsNull()
```

Comment:
You can remove this test - it's redundant since `IsSuccess` has static checking associated with its state, which you test above.

Reply:
Test removed.